stdlib/2/glob.pyi

```diff
@@ -1,4 +1,8 @@
-from typing import List, Iterator, AnyStr
+from typing import List, Iterator, Union
 
-def glob(pathname: AnyStr) -> List[AnyStr]: ...
-def iglob(pathname: AnyStr) -> Iterator[AnyStr]: ...
+_string = Union[str, unicode]
+
+def glob(pathname: _string) -> List[_string]: ...
```

Comment:
I'm unhappy with this. Returning a list of unions is a rather big price to pay. And AFAICT glob() really does return a list of unicodes when the pattern is unicode, and a list of str when the pattern is str. @ambv maybe you can roll this back until we've sorted out what to do?